src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -47,7 +48,8 @@ protected void configure() {
     bind(ClientRequestAdapterFactory.class).in(SINGLETON);
     bind(ClientResponseAdapterFactory.class).in(SINGLETON);
 
-    bind(ZipkinHttpClient.class).to(ZipkinHttpClientImpl.class);
+
+    bind(HttpClient.class).annotatedWith(Zipkin.class).to(ZipkinHttpClientImpl.class);
```

Comment:
Since `ZipkinHttpClientImpl` don't we need a provider in here as well so that an instance can be created?  Otherwise, how does it know which impls to use for the interface constructor args?